BUILDING.txt

```diff
@@ -163,13 +163,13 @@ Maven build goals:
  YARN Application Timeline Service V2 build options:
 
    YARN Timeline Service v.2 chooses Apache HBase as the primary backing storage. The supported
-   versions of Apache HBase are 1.2.6 (default) and 2.0.0-beta1.
+   versions of Apache HBase are 1.7.1 (default) and 2.2.4.
 
-  * HBase 1.2.6 is used by default to build Hadoop. The official releases are ready to use if you
-    plan on running Timeline Service v2 with HBase 1.2.6.
+  * HBase 1.7.1 is used by default to build Hadoop. The official releases are ready to use if you
+    plan on running Timeline Service v2 with HBase 1.7.1.
 
-  * Use -Dhbase.profile=2.0 to build Hadoop with HBase 2.0.0-beta1. Provide this option if you plan
-    on running Timeline Service v2 with HBase 2.0.
+  * Use -Dhbase.profile=2.2 to build Hadoop with HBase 2.2.4. Provide this option if you plan
```

Comment:
It must be `-Dhbase.profile=2.0` if we keep current hadoop-project/pom.xml.
https://github.com/apache/hadoop/blob/8bf72346a59c2eb785fb2a8e55c0246a4f158646/hadoop-project/pom.xml#L2590-L2595